### DISCUSSION TITLE
Hidden Setting/Option :: "JSONmaxParallelWorkers" :: Changing Default of "8" to "1" Is Not Behaving as Expected?

### DISCUSSION THREAD

**Discussion by touwys** [gildas-lormeau/SingleFile#1637] [General]
I changed the `maxParallelWorkers` setting from the default "8" to "1", expecting to be able to save all the open browser tabs in the exact sequence they are sorted from start to finish. This does not happen, since the tabs are still getting saved in random order. Is there perhaps another way to address this?

Thank you.

**Comment by gildas-lormeau** [comment]
It looks like it's a bug. What browser are you using?

**Comment by touwys** [comment]
Firefox 133* / Windows 10 x64
